### PR TITLE
SIG: add sig-scheduling

### PR DESCRIPTION
**Pull request by zhangjinpeng1987**
Signed-off-by: zhangjinpeng1987 <zhangjinpeng@pingcap.com>

SIG: add sig-scheduling

### CODE REVIEW

File: special-interest-groups/sig-scheduling/member-list.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+
+**NOTE**:
+
+* This member list is updated on a bi-weekly basis.
+* Last update time: 2020-02-21
+
+## Tech Leads
+
+Leads:
+* [nolouch](https://github.com/nolouch), [PingCAP](https://pingcap.com/en/)
```
Comment (nolouch) [inline special-interest-groups/sig-scheduling/member-list.md]:
```suggestion
* [nolouch](https://github.com/nolouch), [PingCAP](https://pingcap.com/en/)
* [rleungx](https://github.com/rleungx), [PingCAP](https://pingcap.com/en/)
```

Comment (nolouch) [inline special-interest-groups/sig-scheduling/member-list.md]:
```suggestion
* [nolouch](https://github.com/nolouch), [PingCAP](https://pingcap.com/en/)
* [rleungx](https://github.com/rleungx), [PingCAP](https://pingcap.com/en/)
```

File: special-interest-groups/sig-scheduling/roles-and-organization-management.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
+# SIG Scheduling Roles and Organization Management
+
+## Join and Promotion
+
+### Join
+
+The minimal requirement for contributors to join the Scheduling SIG is to have the **Active Contributor** community role in the PD repository.
```
Comment (zz-jason) [inline special-interest-groups/sig-scheduling/roles-and-organization-management.md]: Once a people contributes to a part of scheduling system(triage issue, merged a pull request, etc.), he's joined the SIG automatically.

Comment (zhangjinpeng1987) [reply special-interest-groups/sig-scheduling/roles-and-organization-management.md]: done

**Pull request merged**